cmake/external/cares.cmake

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(ExternalProject)
+
+ExternalProject_Add(
+  cares
```

Comment:
I don't know how much you *care*, but c-ares is more often stylized with a dash. ("c-ares" vs "cares").

Reply:
I almost named this branch "wilhuff/cares". It turns out, I do!

Renamed the target and the file.